### DISCUSSION TITLE
No module named

### DISCUSSION THREAD

**Discussion by whode** [mitmproxy/mitmproxy#5789] [General]
Hi. I want to import module bs4 (which is installed) in my script, but when I run mitmproxy I get an error: No module named 'bs4'. Can you please tell me why this happens?

**Comment by mhils** [comment]
https://docs.mitmproxy.org/stable/overview-installation/#installation-from-the-python-package-index-pypi

The binaries bring their own environment.

**Reply by whode** [reply]
So I have to uninstall what was installed with the binary and install it again via pip?

**Reply by mhils** [reply]
Yes, the Python environment for the binaries is frozen/independent so that they work irrespective of your system's Python config.
